### ISSUE TITLE
Unable to reproduce the results in Table 1

### ISSUE THREAD

**Issue by GodXuxilie** [PKU-ML/DynACL#1]
Dear Authors,

Thanks for your great work on improving ACL and sharing the code! I encountered some issues when reproducing the results.

I downloaded your pre-trained weights and used the following scripts to conduct SLF:
```
python test_LF.py --experiment SLF_cifar10/DynACL_publish --checkpoint ~/DYNACL/DynACL_CIFAR10.pt --dataset cifar10 --cvt_state_dict --bnNameCnt 1 --evaluation_mode SLF

python test_LF.py --experiment SLF_cifar100/DynACL_cifar100_publish --checkpoint ~/DYNACL/DynACL_CIFAR100.pt --dataset cifar100 --cvt_state_dict --bnNameCnt 1 --evaluation_mode SLF

python test_LF.py --experiment SLF_stl10/DynACL_stl10_publish --checkpoint ~/DYNACL/DynACL_STL10.pt --dataset stl10 --cvt_state_dict --bnNameCnt 1 --evaluation_mode SLF
```

I got the following results from the file ```robustness_result.txt```  (as far as I understand, ```initial accuracy``` stands for ```natural test accuracy```):
CIFAR10: natural test accuracy is 75.39% and AutoAttack is 45.05%;
CIFAR100: natural test accuracy is 36.64% and AutoAttack is 18.66%;
STL10: natural test accuracy is 61.79% and AutoAttack is 44.79%;

Unfortunately, my reproduced results, especially **natural test accuracy**, are **lower** than those reported in Table 1.

Could you please help me figure out the reasons and guide me to reproduce your results?

Thank you!

Best wishes,
Xilie

**Comment by Red-Fairy** [comment]
Xilie,

I'm sorry for not getting back to you sooner, as we didn't receive any email notification from GitHub. I guess that you are using the checkpoint for DYNACL instead of DYNACL++, which has an additional post-processing stage. I find your score on CIFAR-10 matches that of DYNACL. For CIFAR-100 and STL-10, you should change the initial learning rate to obtain a higher score.

Best,
Rundong

**Comment by GodXuxilie** [comment]
Hi Rundong,

Many thanks for your reply. Could you please recommend the hyper-parameter for CIFAR-100 and STL10? Currently, I found the initial learning rate is 0.01 for CIFAR-10. Would the initial learning rate become larger for CIFAR-100 and STL10?

Best wishes,
Xilie

**Comment by Red-Fairy** [comment]
Yes, I use a larger learning rate for the larger datasets. Specifically, I use 0.05 for CIFAR-100 and 0.1 for STL-10.

**Comment by GodXuxilie** [comment]
Thank you! I successfully reproduced the results!
